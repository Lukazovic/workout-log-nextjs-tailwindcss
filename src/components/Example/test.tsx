import { render, screen } from '@testing-library/react'

import Example from '.'

describe('<Example />', () => {
  it('should render Example heading', () => {
    const { container } = render(<Example />)

    expect(
      screen.getByRole('heading', { name: /example/i })
    ).toBeInTheDocument()

    expect(
      screen.getByText(/A component example with TailwindCSS/i)
    ).toBeInTheDocument()

    expect(
      screen.getByRole('img', { name: /tailwindcss logo/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
