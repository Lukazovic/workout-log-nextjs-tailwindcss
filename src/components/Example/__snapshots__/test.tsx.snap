// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Example /> should render Example heading 1`] = `
<div
  class="w-full h-screen bg-primary-900 pt-4 m-0 flex flex-col items-center justify-center"
>
  <h1
    class="text-3xl font-bold text-white"
  >
    Example
  </h1>
  <p
    class="text-white mt-6"
  >
    A component example with TailwindCSS
  </p>
  <img
    alt="Tailwindcss Logo"
    class="w-80 h-80"
    src="/tailwindcss.svg"
  />
</div>
`;
